WordPress/Resources/it.lproj/Localizable.strings

```diff
@@ -7456,7 +7456,7 @@ translators: %s: Select control option value e.g: \"Auto, 25%\". */
 "Tap [bookmark-outline] to save a post to your list." = "Tocca [bookmark-outline] per salvare un articolo nel tuo elenco.";
 
 /* Subtitle instructing the user to tap the dismiss button to leave the log in flow */
-"Tap dismiss and head back to your web browser to continue." = "Tocca Ignora e torna al tuo browser web per continuare.";
+"Tap dismiss and head back to your web browser to continue." = "Tocca Respinto e torna al tuo browser web per continuare.";
```

Comment:
This is the fix for the issue @tiagomar picked up in the QR code login:

<img width="459" alt="image" src="https://user-images.githubusercontent.com/1218433/177968390-6db5c778-0978-45c5-aa9b-b27713f55017.png">